python/cudf/cudf/core/_compat.py

```diff
@@ -4,9 +4,6 @@
 from packaging import version
 
 PANDAS_VERSION = version.parse(pd.__version__)
-PANDAS_GE_110 = PANDAS_VERSION >= version.parse("1.1")
-PANDAS_GE_120 = PANDAS_VERSION >= version.parse("1.2")
-PANDAS_LE_122 = PANDAS_VERSION <= version.parse("1.2.2")
 PANDAS_GE_130 = PANDAS_VERSION >= version.parse("1.3.0")
```

Comment:
`PANDAS_GE_130` is always true, right?

Reply:
Yes (so there might be more conditions to cull!)

Reply:
Yup, good catch. Cleaned it up